build-logic/convention/src/main/kotlin/AndroidHiltConventionPlugin.kt

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.artifacts.VersionCatalogsExtension
+import org.gradle.kotlin.dsl.dependencies
+import org.gradle.kotlin.dsl.getByType
+
+class AndroidHiltConventionPlugin : Plugin<Project> {
+    override fun apply(target: Project) {
+        with(target) {
+            with(pluginManager) {
+                apply("dagger.hilt.android.plugin")
+                apply("org.jetbrains.kotlin.kapt")
+            }
+
+            val libs = extensions.getByType<VersionCatalogsExtension>().named("libs")
```

Comment:
You can create a util for **libs** which can be reused across all convention plugins e.g [nowinandroid project extensions](https://github.com/android/nowinandroid/blob/main/build-logic/convention/src/main/kotlin/com/google/samples/apps/nowinandroid/ProjectExtensions.kt), so that we don't declare everytime we need it e.g in [AndroidRoomConventionPlugin](https://github.com/droidconKE/droidconKE2023Android/blob/56c4cd06fa75f6dd6aea4266af5d87d468d3f24d/build-logic/convention/src/main/kotlin/AndroidRoomConventionPlugin.kt#L42)

Reply:
I actually thought about this, thanks for the reminder.

Reply:
Good point